include/inexor/vulkan-renderer/wrapper/command_pool.hpp

```diff
@@ -26,10 +26,14 @@ class CommandPool {
     CommandPool(const VkDevice device, const std::uint32_t queue_family_index);
 
     ~CommandPool();
-
+    
     [[nodiscard]] VkCommandPool get() const {
         return command_pool;
     }
+
+    [[nodiscard]] auto get_ptr() const {
```

Comment:
Should be explicit with `VkCommandPool *`

Reply:
Right.

Reply:
Done.